public/pages/DetectorResults/containers/AnomalyResults.tsx

```diff
@@ -332,6 +346,35 @@ export function AnomalyResults(props: AnomalyResultsProps) {
                       style={{ marginBottom: '20px' }}
                     >
                       {getCalloutContent()}
+                      {isDetectorInitializing && detector.initProgress ? (
+                        <div>
+                          <EuiFlexGroup alignItems="center">
+                            <EuiFlexItem
+                              style={{ maxWidth: '20px', marginRight: '5px' }}
+                            >
+                              <EuiText>
+                                {
+                                  //@ts-ignore
+                                  detector.initProgress.percentageStr
+                                }
+                              </EuiText>
+                            </EuiFlexItem>
+                            <EuiFlexItem>
+                              <EuiProgress
+                                //@ts-ignore
+                                value={detector.initProgress.percentageStr.replace(
+                                  '%',
+                                  ''
+                                )}
+                                max={100}
+                                color="primary"
+                                size="xs"
+                              />
+                            </EuiFlexItem>
+                          </EuiFlexGroup>
+                          <EuiSpacer size="l" />
+                        </div>
+                      ) : null}
                       <EuiButton
                         onClick={props.onSwitchToConfiguration}
```

Comment:
Why do we have a switch to configuration button when the configuration tab is only a finger click away?

Reply:
I know this was just part of the existing approved UI, but agreed that it doesn't really seem necessary with the tab right above. Guess it was for quickly navigating to config page since a config change must have occurred for initialization process to begin again / for callout to appear again.

Reply:
Yeah, this already exists in UX design since long time ago. This configuration button is shown in callout because all the messages in callout mentions detector configuration, and such button can direct user quickly to detector config page .